src/main/scala/nl/knaw/dans/easy/solr/EasyUpdateSolrIndex.scala

```diff
@@ -50,9 +50,9 @@ object EasyUpdateSolrIndex {
     if (settings.datasetQuery.isDefined)
       settings.datasetQuery.get.foreach(datasetsFromQuery(_))
     else if (settings.input.isDefined)
-      readLines(new FileInputStream(settings.input.get)).asScala.foreach(execute)
+      readLines(new FileInputStream(settings.input.get)).asScala.foreach(id => execute(Seq(id)))
```

Comment:
TODO
* change into `execute(readLines(new FileInputStream(settings.input.get)).asScala)`
* apply the batch size
* update help/readme on the new meaning of the batch size